types/tonic-ui__react-icons/scripts/generate-definitions.ts

```diff
@@ -0,0 +1,45 @@
+import * as tmicon from "@trendmicro/tmicon";
```

Comment:
This is a ts file; how are you intending for it to be run?

Reply:
@jakebailey I need to generate type definitions for all icons from the `@trendmicro/tmicon` package. This file would run using `pnpm exec ts-node sripts/generate-definitions.ts` after making changes to the templates in `src` and it would act as a "build script".

Reply:
It is only necessary for the development of this typing package and would not be included in the package itself.

Reply:
You may want to look at other `scripts` dirs like those in `node/scripts`; typically these scripts dirs have their own `package.json` with some info on how to run it.

Reply:
Thank you!